client/my-sites/checkout/checkout/web-payment-box.jsx

```diff
@@ -426,7 +426,7 @@ export class WebPaymentBox extends React.Component {
 					button = (
 						<button
 							type="submit"
-							onClick={ ( event ) => this.submit( paymentMethod, event ) }
+							onClick={ event => this.submit( paymentMethod, event ) }
```

Comment:
Just FYI: We used to have a linting rule against passing lambda functions as props in concerns of potential performance implication describing in https://medium.com/@muralikv/do-not-use-anonymous-functions-or-bind-in-component-props-8e835e1cddec and https://reactjs.org/docs/faq-functions.html#is-it-ok-to-use-arrow-functions-in-render-methods.

However, since the linting rule has gone, I think that means we don't really care about it anymore. Thus this is just an FYI :)

Reply:
Confirmed :)
https://github.com/Automattic/eslint-config-wpcalypso/pull/18